src/map/status.cpp

```diff
@@ -11444,7 +11459,11 @@ int status_change_start(struct block_list* src, struct block_list* bl,enum sc_ty
 			// val4 signals autoprovoke.
 			break;
 		case SC_AVOID:
```

Comment:
If possible update SC_AVOID in /doc/status_change.txt the whole description is missing anyway.